sphinx/types.rst

```diff
@@ -1,104 +1,170 @@
+data with the "meter" key. Users can set each of the following "Field Name" keys with their own
+data's column names in their SCADA data, plus the "freq" field. Users just have to ensure that the
+columns are already using the specified units, and that each column is already using the listed
+data type or can be converted to that type.
+
+==================== ==================================   =============================
+ Field Name           Data Type (MeterMetaData.dtypes)     Units (MeterMetaData.units)
+==================== ==================================   =============================
+ time                 datetime64[ns]                       datetime64[ns]
+ power                float                                kW
+ energy               float                                kWh
+==================== ==================================   =============================
+
+Tower
+=====
+
+:py:attr:`PlantData.tower` is configured by the :py:class:`openoa.plant.TowerMetaData` class, which is set in the configuration
+data with the "tower" key. Users can set each of the following "Field Name" keys with their own
+data's column names in their met tower data, plus the "freq" field. Users just have to ensure that
+the columns are already using the specified units, and that each column is already using the listed
+data type or can be converted to that type.
+
+==================== ==================================   =============================
+ Field Name           Data Type (TowerMetaData.dtypes)     Units (TowerMetaData.units)
+==================== ==================================   =============================
+ time                 datetime64[ns]                       datetime64[ns]
+ id                   string                               None
+==================== ==================================   =============================
+
+Curtail
+=======
+
+:py:attr:`PlantData.curtail` is configured by the :py:class:`openoa.plant.CurtailMetaData` class, which is set in the configuration
+data with the "curtail" key. Users can set each of the following "Field Name" keys with their own
+data's column names in their curtailment data, plus the "freq" field. Users just have to ensure that
+the columns are already using the specified units, and that each column is already using the listed
+data type or can be converted to that type.
+
+==================== ====================================   ===============================
+ Field Name           Data Type (CurtailMetaData.dtypes)     Units (CurtailMetaData.units)
+==================== ====================================   ===============================
+ time                 datetime64[ns]                         datetime64[ns]
+ curtailment          float                                  percent
+ availability         float                                  percent
```

Comment:
curtailment and availability should have the units of energy ("kwh"), not percent. The AEP analysis method assumes the units are kwh, so I think this is just a documentation fix.

Reply:
I've double checked on this and it looks we assume it's all in GWh wherever it's used in the code, so I've updated the metadata class and this piece of documentation to amend this.